website/docs/d/function_app_host_keys.html.markdown

```diff
@@ -19,7 +19,8 @@ data "azurerm_function_app_host_keys" "example" {
 }
 ```
 
-~> **Note:** The unencrypted value of FunctionKeys, MasterKey and SystemKeys will be stored in the raw state as plain-text.
+~> **Note:** All arguments including the secret value will be stored in the raw state as plain-text.
+[Read more about sensitive data in state](/docs/state/sensitive-data.html).asterKey and SystemKeys will be stored in the raw state as plain-text.
```

Comment:
Can we modify the wording here a bit:

```suggestion
~> **Note:** All arguments including the secret value will be stored in the raw state as plain-text, including `default_function_key` and `master_key`. [Read more about sensitive data in state](/docs/state/sensitive-data.html).
```

Reply:
![image](https://user-images.githubusercontent.com/42009880/92221509-e35cb000-ee9d-11ea-9e52-0fcd6c77ecac.png)

but I have no problems with changing it if you still wish so